spec/unit/hanami/helpers/html_helper/html_builder_spec.rb

```diff
@@ -245,6 +245,13 @@
       result = @builder.input('required' => true, 'value' => 'Title "book"', 'something' => 'bar').to_s
       expect(result).to eq('<input required="required" value="Title "book"" something="bar">')
     end
+
+    # rubocop:disable Style/SymbolArray
+    it 'renders attribute with array value as joined with string' do
+      result = @builder.input('class' => [:ui, :form]).to_s
+      expect(result).to eq('<input class="ui form">')
+    end
```

Comment:
@cllns I'd add another spec also to cover `HtmlNode`. For now the implementation of this PR brings the benefits of this enhancement because `HtmlNode` inherits from `EmptyHtmlNode`, but if that hierarchy changes, we may loose track of this feature for `HtmlNode`, unless we add a test.

Reply:
:D good call. It actually didn't work for non-empty HTML nodes 😅 . `HtmlNode` doesn't call `super`, so I had to re-do the call to `htmlified_array_attributes`.